backend/corpora/common/corpora_orm.py

```diff
@@ -158,7 +158,9 @@ class DbProject(Base):
     created_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
     updated_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
 
-    user = relationship("DbUser")
+    user = relationship("DbUser", uselist=False)
```

Comment:
Are we still committing to back_populating everywhere for consistency?

Reply:
Ditto for links below

Reply:
yeah let me add that in.